Claudinee11/My-Dairy
# Ft edit 168234152

What does this PR do?
- added logo on the signup page
- makes the logo as a link
- edit forget  password form
Description of Task to be completed
A user should be able to forget the password using a form and on clicking on submit the redirected to forget password page.
How should this be manually tested?
- clone the repo and switch to the branch ft-ft-edit-168234152
- enter details in the field and click submit
you should be successfully redirected to change.html
What are the relevant pivotal tracker stories?
- 1682344152
screenshot:
![image](https://user-images.githubusercontent.com/50420731/64288450-03c20800-cfc7-11e9-8e6a-f69c26ee7acb.png)
